amundsen_application/static/js/ducks/allTags/reducer.ts

```diff
@@ -31,7 +31,11 @@ export default function reducer(state: AllTagsReducerState = initialState, actio
     case GetAllTags.FAILURE:
       return initialState;
     case GetAllTags.SUCCESS:
-      return { ...state, allTags: (<GetAllTagsResponse>action).payload.tags, isLoading: false };
+      return {
```

Comment:
nit (No changes requested): I get why this simplifies this current use case, but I'd encourage us to continue being conscious of the fact that logic which leverages the fact that the response payload and redux state happen to be the same can create extra work as the application scales. Apis can potentially be extended to include new information, and application state can potentially be restructured for performance/best practices. If that day comes with the code as is sure it would be a quick fix, but if we ended up building further tag logic and helper methods with this assumption we'd have a lot of extra work to do.

Reply:
I see what you mean. This made more sense when `curatedTags` and `otherTags` were also part of the payload, but after removing them it might make more sense to explicitly list out the `allTags` field.